.teamcity/src/main/kotlin/promotion/PublishGradleDistribution.kt

```diff
@@ -34,26 +34,24 @@ abstract class PublishGradleDistribution(
 
     init {
         artifactRules = """
-        incoming-build-receipt/build-receipt.properties => incoming-build-receipt
         **/build/git-checkout/subprojects/base-services/build/generated-resources/build-receipt/org/gradle/build-receipt.properties
         **/build/distributions/*.zip => promote-build-distributions
         **/build/website-checkout/data/releases.xml
         **/build/git-checkout/build/reports/integTest/** => distribution-tests
         **/smoke-tests/build/reports/tests/** => post-smoke-tests
     """.trimIndent()
 
+        val absoluteId = RelativeId("Check_Stage_${this@PublishGradleDistribution.triggerName}_Trigger")
         steps {
             gradleWrapper {
                 name = "Promote"
                 tasks = task
-                gradleParams = """-PuseBuildReceipt $extraParameters "-PgitUserName=$gitUserName" "-PgitUserEmail=$gitUserEmail"  """
+                gradleParams = """-PcommitId=%dep.Gradle_${promotedBranch.capitalize()}_Check_Stage_${this@PublishGradleDistribution.triggerName}_Trigger.build.vcs.number% $extraParameters "-PgitUserName=$gitUserName" "-PgitUserEmail=$gitUserEmail"  """
```

Comment:
I don't think we should use `promotedBranch`, but instead use the `RelativeId` here.

Reply:
How can we reference an absolute parameter via `RelativeId`? It has no method like `toAbosolute()`.

Reply:
IIRC, `RelativeId.toString()` gives you the right string. We are already doing this here: https://github.com/gradle/gradle/blob/608bee705b63d5b48df33df9df607a0e7bdbdeab/.teamcity/src/main/kotlin/promotion/PublishBranchSnapshotFromQuickFeedback.kt#L38